db/schema.rb

```diff
@@ -362,10 +395,19 @@
     t.datetime "locked_at"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.boolean  "isActive",                                   default: true
```

Comment:
You have the columns duplicated, maybe because you renamed them in the migration without first rolling back.